src/platform-aws.c

```diff
@@ -69,27 +70,7 @@ static struct ec2_platform_data platform_data_map[] = {
 		.domain_per_thread = 0,
 	},
 	{
-		.name = "p5.48xlarge",
-		.topology = NULL,
-		.default_dup_conns = 0,
-		.latency = 75.0,
-		.gdr_required = true,
-		.net_flush_required = false,
-		.default_protocol = "RDMA",
-		.domain_per_thread = 0,
-	},
-	{
-		.name = "p5e.48xlarge",
-		.topology = NULL,
-		.default_dup_conns = 0,
-		.latency = 75.0,
-		.gdr_required = true,
-		.net_flush_required = false,
-		.default_protocol = "RDMA",
-		.domain_per_thread = 0,
-	},
-	{
-		.name = "p5en.48xlarge",
+		.name = "p5.*",
```

Comment:
I know the odds of a collision are low, but probably want to anchor all these patterns with a leading `^`?

Reply:
yup.  fixed all to anchor the start, and the exact match also anchor the end, because why not?